src/patternfly/components/EmptyState/empty-state.scss

```diff
@@ -1,64 +1,54 @@
 .pf-c-empty-state {
   --pf-c-empty-state--Padding: var(--pf-global--spacer--xl);
+  --pf-c-empty-state__content--MaxWidth: none;
   --pf-c-empty-state__icon--MarginBottom: var(--pf-global--spacer--lg);
   --pf-c-empty-state__icon--FontSize: var(--pf-global--icon--FontSize--xl);
   --pf-c-empty-state__icon--Color: var(--pf-global--icon--Color--light);
   --pf-c-empty-state__body--MarginTop: var(--pf-global--spacer--md);
   --pf-c-empty-state__body--Color: var(--pf-global--Color--200);
-  --pf-c-empty-state--c-button--MarginTop: var(--pf-global--spacer--xl);
-  --pf-c-empty-state--c-button__secondary--MarginTop: var(--pf-global--spacer--sm);
+  --pf-c-empty-state__primary--MarginTop: var(--pf-global--spacer--xl);
+  --pf-c-empty-state__primary--secondary--MarginTop: var(--pf-global--spacer--sm);
```

Comment:
```suggestion
  --pf-c-empty-state__secondary--MarginTop: var(--pf-global--spacer--sm);
```

Reply:
Not sure what you mean - this is for when the secondary element follows the primary element/button. And `--pf-c-empty-state__secondary--MarginTop` is already defined in the line below

Reply:
Oh, I see what you're doing!